src/client/components/Navigation/NavGeo/Sections/Background/Background.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const Background: React.FC = () => {
+  return (
+    <div>
+      <span>Background content</span>
+    </div>
+  )
+}
+
+export default Background
```

Comment:
I am thinking of adding the rest of the sections (forest, protected, and burned area layers) here under `src/client/components/Navigation/NavGeo/Sections/<Section name>`, but it's open for suggestions.